_install-and-configure/configuring-opensearch/security-settings.md

```diff
@@ -122,6 +122,41 @@ The Security plugin supports the following expert-level settings:
 
 - `plugins.security.check_snapshot_restore_write_privileges` (Static): Enforces write privilege evaluation when creating snapshots. Default is `true`.
 
+If you change any of the following password hashing properties, you must rehash all internal passwords to ensure compatibility and security.
+{: .warning}
+
+- `plugins.security.password.hashing.algorithm`: (Static): Specifies the password hashing algorithm to use.
+
+  Valid values are:
+  
+  - `BCrypt` (Default)
+  - `PBKDF2` 
+
+- `plugins.security.password.hashing.bcrypt.rounds` (Static): Specifies the number of rounds to use for password hashing with BCrypt. Valid values are from `4` to `31`, inclusive. Default is `12`.
```

Comment:
What do we mean by inclusive?

```suggestion
- `plugins.security.password.hashing.bcrypt.rounds` (Static): Specifies the number of rounds to use for password hashing with `BCrypt`. Valid values are a range from `4` to `31`, inclusive. Default is `12`.
```

Reply:
Just meant that the range includes 4 and 31 not just the numbers between them

Reply:
Thoughts on "Valid values range from `4` to `31`, inclusive." ?

Reply:
@dancristiancecoi Alternatively, feel free to use the math notation for ranges (`[4, 31]`).

Reply:
Updated the suggestion accordingly.

Reply:
Thanks a lot! Just a tiny thing @Naarcha-AWS, was that space removed on purpose?